javascript/yarn.lock

```diff
@@ -2,6 +2,30 @@
 # yarn lockfile v1
 
 
+"@stimulus/core@^1.1.1":
```

Comment:
Shouldn't this reflect v2.0.0?

Reply:
Is Stimulus v2 imminent?

Reply:
@pjforde1978 the changes in this PR that Nate and I paired on are breaking so I’d say yes though it’s obviously ultimately up to Nate.

Reply:
That dep is for Stimulus itself rather than StimulusReflex.